.github/workflows/android.yml

```diff
@@ -34,10 +34,13 @@ jobs:
       run: chmod +x gradlew
 
     - name: Build with Gradle
-      run: ./gradlew build
+      run: ./gradlew assemble
```

Comment:
Why did we change this?

Reply:
`build` depends on `check`, which means `build` already runs tests. This is not the behaviour we need as we run unit tests separately. Consequently, if the `tests` fail then the `build` also fails.

Reference: https://stackoverflow.com/a/61878404

Reply:
Sounds good! Could we create another PR for it and let this only focus on ktlint?

Reply:
Umm, sure. Would it be okay if I keep this as a separate commit?

Reply:
Yes :)

Reply:
Fixed in latest PR. Will create separate PR for this